src/Shared/TaskHostConfiguration.cs

```diff
@@ -458,6 +464,13 @@ internal static INodePacket FactoryForDeserialization(ITranslator translator)
         {
             TaskHostConfiguration configuration = new TaskHostConfiguration();
             configuration.Translate(translator);
+#if FEATURE_APPDOMAIN
+            if (configuration._appDomainConfigBytes != null)
+            {
+                configuration._appDomainSetup = new AppDomainSetup();
```

Comment:
This should be executed even if config bytrs are null.
(applies to same code above as well)